application-accelerator/creating-accelerators/custom-types.hbs.md

```diff
@@ -0,0 +1,163 @@
+# Using Custom Types
+Application Accelerators can declare new types to be used for options declaration,
+in addition to the built-in types `string`, `number` and `boolean`.
+
+## Introducing the Types Section
+
+The `accelerator.yaml` file syntax supports an entry named `types` inside the top-level
+`accelerator` section, allowing the definition of custom types, with support for
+`struct` types, like so:
+```yaml
+accelerator:
+  options:
+    ...
+  types:
+    - name: Task
+      struct:
+        - name: title
+          dataType: string
+          label: Title
+          description: A sample title
+        - name: details
+          label: Task details
+          description: Enter the task details
+        - name: done
+          dataType: boolean
+          label: Done?
+          defaultValue: false
+```
+
+The `struct` type definition is syntactically equivalent to **a sequence (hence ordered) of
+option definitions**.
+
+A type name MUST be a `CapitalizedName`.
+
+## Interaction with UIs and Values Submission
+
+The effect of the previous example is that a new _type_ then becomes available
+(in addition to the built-in `string`, `number`, `boolean` and sequences thereof)
+for the `dataType` property of any option:
+```yaml
+accelerator:
+  options:
+    - name: myTask
+      dataType: Task
+  types:
+    ...
+```
+whereby UIs would render something similar to the following:
+![SimpleTask](simple-task.png)
```

Comment:
This graphic has a mixture of languages. There is a different language under Enter task details
It would be better to get an English only graphic if you could supply one